src/Orchard.Cms.Web/Modules/Orchard.OpenId/Startup.cs

```diff
@@ -44,18 +43,39 @@ public class Startup : StartupBase
         public override void Configure(IApplicationBuilder builder, IRouteBuilder routes, IServiceProvider serviceProvider)
         {
             var openIdService = serviceProvider.GetService<IOpenIdService>();
-            var openIdSettings = openIdService.GetOpenIdSettingsAsync().Result;
-            if (!openIdService.IsValidOpenIdSettings(openIdSettings))
+            var settings = openIdService.GetOpenIdSettingsAsync().Result;
```

Comment:
Yeah, I agree it's better (it doesn't wraps the exception in an `AggreateException`). I'll fix that as part of this PR.